edgefirewall.yaml

```diff
@@ -0,0 +1,333 @@
+openapi: 3.0.0
+info:
+  title: Azion API
+  version: "3.0"
+servers:
+- url: https://api.azionapi.net/
+paths:
+  /edge_firewall:
```

Comment:
Should this file be on this PR? It's not mentioned on the PR title, and it's already being added on another PR: https://github.com/aziontech/azionapi-openapi/pull/33